src/renderer/components/drawer/drawer-title.tsx

```diff
@@ -9,7 +9,12 @@ import { cssNames } from "../../utils";
 
 export interface DrawerTitleProps {
   className?: string;
-  children: React.ReactNode;
+  children?: React.ReactNode;
+
+  /**
+   * @deprecated Prefer passing the value as `children`
+   */
+  title?: React.ReactNode;
```

Comment:
Why it was deprecated and `props.children` preferred?

Reply:
Mostly because the react docs imply that a container like `DrawerTitle` should prefer the children props:

> Some components don’t know their children ahead of time. This is especially common for components like Sidebar or Dialog that represent generic “boxes”.
> 
> We recommend that such components use the special children prop to pass children elements directly into their output: